### PR TITLE
BEI-34 Connect The Database

### PR DESCRIPTION
**Pull request by RICHYDOS**
WHY THIS PR?
- To ensure that our server is connected to the database when making and receiving requests.

WHAT DOES THIS PR DO?
- Connects the server to the database

HOW CAN THIS BE TESTED
- You should see "Connected to Database" when running/starting the server

THINGS YOU MAY WANT THE REVIEWER TO NOTE
N/A

### CODE REVIEW

File: src/server.ts [hunk 1]
```diff
@@ -1,17 +1,20 @@
...
 
-import app from './app'
-import { logger } from './core/util/logger'
+sequelize.sync().then(() => {
+  console.log('Connected to the Database');
+});
 
-const startServer =async(): Promise<void>=>{
-  app.listen(process.env.APP_PORT, ()=>{
+const startServer = async (): Promise<void>=>{
```
Comment (LucianaDera) [inline src/server.ts:10]: Any reason for the added spaces

File: src/server.ts [hunk 2]
```diff
@@ -1,17 +1,20 @@
...
-dotevn.config()
+import { setting } from '../src/core/config/application';
+import app from './app';
+import { logger } from './core/util/logger';
+import { sequelize } from './core/database/sequelize';
 
-import app from './app'
-import { logger } from './core/util/logger'
+sequelize.sync().then(() => {
+  console.log('Connected to the Database');
```
Comment (LucianaDera) [inline src/server.ts:7]: You can use the logger her

File: .env.example [hunk 3]
```diff
@@ -1,5 +1,6 @@
...
-MY_SQL_PASS=
-HOST=
-SHOW_APPLICATION_LOGS=
\ No newline at end of file
+MYSQL_DB=
+MYSQL_USER=
+MYSQL_PASS=
+MYSQL_HOST=
+SHOW_APPLICATION_LOGS=
+APP_PORT=
```
Comment (LucianaDera) [inline .env.example:6]: Add an extra line

**Review by LucianaDera — CHANGES_REQUESTED** [review]

File: src/server.ts (hunk 1 above)
Comment (RICHYDOS) [reply src/server.ts:10]: It was having issues with eslint

File: .env.example (hunk 3 above)
Comment (RICHYDOS) [reply .env.example:6]: Done

**Review by LucianaDera — APPROVED** [review]

**Pull request merged**